scripts/airgap.sh

```diff
@@ -0,0 +1,195 @@
+: ${POLICIES_TAR:=kubewarden-policies.tar.gz}
+
+kwctl_args=${KWCTL_SOURCES:+--sources-path $KWCTL_SOURCES}
+tmpfile=$(mktemp /tmp/kubewarden-airgap.XXXXXX)
+
+# ==================================================================================================
+# Functions
+
+# Print green colored message
+function step() { printf -- "\e[32m${*}\e[0m\n"; }
+
+# Get latest policiy and image lists from artifacthub
+# Use text files from kubewarden-defaults chart
+function pull_latest_lists() {
+   tag=$(helm search hub kubewarden-defaults -o json | jq -er '.[] | select(.repository.url=="https://charts.kubewarden.io").version')
+   assets_urls=$(curl -sL https://api.github.com/repos/kubewarden/helm-charts/releases/tags/kubewarden-defaults-$tag | jq -er '.assets[].browser_download_url')
+   imagelist_url=$(grep 'images.txt$' <<< "$assets_urls")
+   policylist_url=$(grep 'policylist.txt$' <<< "$assets_urls")
+}
+
+# Download images & policies
+# If no $source is set use defaults from pull_latest_lists
+function pull() {
+   local source="$1"    # file or url with list of images or policies
+   local target="$2"    # target archive filename
+   local command        # docker for images or kwctl for policies
+
+   [ -f "$target" ] && { echo "File $target already exists, skipping pull."; return; }
+
+   # if source is url download it
+   if [[ "$source" =~ ^https:// ]]; then
+      curl -sL $source -o "$tmpfile"
+      source=$tmpfile
+   fi
+
+   grep -qF '://' $source && command=kwctl || command=docker
+   cat $source | while read line || [[ -n $line ]]; do
+      echo "Pull: $line"
+      $command pull "$line" > /dev/null
+   done
+   echo "Create $target from pulled images"
+   $command save $(cat $source) --output "$target"
+}
+
+# Upload images & policies into $REGISTRY
+# If no $source is set generate lists from archive content
+function push() {
+   local source="$1"     # source list
+   local archive="$2"   # source archive filename
+   local command        # docker or kwctl
+
+   [ ! -f "$archive" ] && { echo "File $archive does not exist, skipping push."; return; }
+   tar -tf $archive | grep manifest.json >/dev/null && command='docker' || command='kwctl'
+
+   if [ -z "$source" ]; then
+      [ $command == 'docker' ] && tar -xOf $archive manifest.json | jq -re '.[].RepoTags[]' > "$tmpfile"
+      [ $command == 'kwctl' ] && tar -tf $archive | sed 's#/#://#' > "$tmpfile"
+      source=$tmpfile
+   fi
+
+   echo "Loading images from $archive"
+   $command load --input "$archive"
+   cat $source | while read image || [[ -n $image ]]; do
+      echo -n "Push: "
+      if [ $command == 'docker' ]; then
+         local target=$REGISTRY/${image#*/}
+         docker tag $image $target
+         docker push -q $target
+      elif [ $command == 'kwctl' ]; then
+         local target=$(sed -E "s#.*://[^/]+#registry://$REGISTRY#" <<< "$image")
+         echo "$target"
+         kwctl push $image $target ${kwctl_args:-} >/dev/null
```

Comment:
```suggestion
         kwctl push $image $target ${kwctl_args:-}
```

I would like to print the list of policies being pushed to be fair.